app/assets/stylesheets/_base.scss

```diff
@@ -61,6 +61,13 @@ select {
   width: 100%;
   height: 100%;
   transform: translate(-50%, -50%);
+
+  &.-absolute {
```

Comment:
I wouldn't add this rules to the global scope but only to the modal. We don't use it anywhere else so let's keep it local.